views/my_packages.package.dt

```diff
@@ -41,6 +41,11 @@ block body
 		form(method="POST", enctype="multipart/form-data", action="#{req.rootDir}my_packages/#{packageName}/delete_logo")
 			button.danger(type="submit") Delete
 
+		form(method="POST", action="#{req.rootDir}my_packages/#{packageName}/set_documentation_url")
+			h3 Documentation URL
+			input(name="documentation_url", value="#{pack[`documentation_url`].get!string}", pattern="https?://.+")
```

Comment:
Missed one name check here (the second "documentation_url").

Reply:
Ah forgot to add `.dt` to the replace command and it still compiled because it's Json :/